dockerfile/cuda11.1.1.dockerfile

```diff
@@ -49,13 +49,15 @@ RUN cd /tmp && \
     tar --extract --file docker.tgz --strip-components 1 --directory /usr/local/bin/ && \
     rm docker.tgz
 
-# Configure SSH
+# Update system config
 RUN mkdir -p /root/.ssh && \
     touch /root/.ssh/authorized_keys && \
     mkdir -p /var/run/sshd && \
     sed -i "s/[# ]*PermitRootLogin prohibit-password/PermitRootLogin yes/" /etc/ssh/sshd_config && \
     sed -i "s/[# ]*Port.*/Port 22/" /etc/ssh/sshd_config && \
-    echo "PermitUserEnvironment yes" >> /etc/ssh/sshd_config
+    echo "PermitUserEnvironment yes" >> /etc/ssh/sshd_config && \
```

Comment:
Whether will have "PermitUserEnvironment yes" in multiple lines?

Reply:
probably if base image changes, but it does not matter, duplicate lines will work fine